settings.gradle

```diff
@@ -1,6 +1,6 @@
 pluginManagement {
     repositories {
-        mavenLocal();
+        mavenLocal()
```

Comment:
Setting mavenLocal at the module level can be confusing. I suggest to remove it from here.

Reply:
Done. Removed mavenLocal()